### ISSUE TITLE
User groups

### ISSUE THREAD

**Issue by wojcik13** [mjnaderi/Sharif-Judge#64]
Hi,
can somebody help me? I don't know how to add users groups, so user A can only see assignment A, and user B can only see assignment B, so that A can't see B. Maybe we could add user categories function or sth?
Kindly Regards

**Comment by truongan** [comment]
There never was a "user groups" feature. You can allow only specific users to submit via assignment interface but that was it I'm afraind.

**Comment by wojcik13** [comment]
yep, but I want to create a function, described above, and i don't know how to do it. Can somebody help me? Kindly Regards c:

**Comment by truongan** [comment]
Write the code yourself is the spirit for opensource project like this
